api/compliance.go

```diff
@@ -106,3 +106,6 @@ type ComplianceViolation struct {
 	Resource string   `json:"resource"`
 	Reasons  []string `json:"reasons"`
 }
+
+// ValidComplianceStatuses is a list of all valid compliance statuses
+var ValidComplianceStatuses = []string{"non-compliant", "requires-manual-assessment", "suppressed", "compliant", "could-not-assess"}
```

Comment:
where did we get the list of valid compliance status? also, could we call this `ValidComplianceStatus` instead. I don't think statuses is a word. 😂

Reply:
It is according to the oxford english dictionary :) 
But I will change it if it is not valid US-English

Reply:
The valid list I pulled from a report. Though I can't guarantee this is all valid reports and I do not know if they are consistent across cloud providers. I will remove this validation from status filter.

Reply:
If it is, please ignore me, what would I know, I'm Mexican 😂. 

For the valid list, I found something inside the platform, I'm pinging you directly over in Slack.